src/store/contacts.js

```diff
@@ -331,9 +331,7 @@ const actions = {
 		validate(contact)
 
 		// Update REV
-		const rev = new ICAL.VCardTime()
-		rev.fromUnixTime(Date.now() / 1000)
-		contact.rev = rev
+		contact.rev = ICAL.VCardTime.now()
```

Comment:
The old code did create a date-time string including the timezone. 

` ICAL.VCardTime.now()` does create a value without a timezone like `REV:20250723T165820`

Don't you think we should keep the timezone?